src/app/interfaces.ts

```diff
@@ -3,4 +3,6 @@ export interface Movie {
   id: string;
   link: string;
   loading?: boolean;
+  statistics?: string;
```

Comment:
czemu statistics? wypisz tu sobie wszystko, czego wymaga zadanie. Powiedzmy, że wymaga liczby wyświetleń i liczby lajków, napisz zatem viewCount: number, likeCount: number.

Reply:
(nie pamitam czego wymaga zadanie)

Reply:
statistics to jak widze w interfejsach cały obiekt, no możesz i tak zrobić ze zapiszesz go sobie tu w całości, ale w taim razie to nie jest typ string tylko Statistics - zdecyduj się ;)